docker-compose.yml

```diff
@@ -20,8 +20,6 @@ services:
       - pgdata:/var/lib/postgresql/data
     networks:
       - main
-    ports:
-      - 5432:5432
```

Comment:
You don't need to keep these, like in the redis services?

Reply:
I found that it didn't have any impact on which port was used when I tested it.